### DISCUSSION TITLE
When I press the back button with the keyboard open, it adds white space at the bottom

### DISCUSSION THREAD

**Discussion by DylanThomasFr** [react-navigation/react-navigation#11190] [Q&A]
https://user-images.githubusercontent.com/48915764/216589917-55351270-98eb-4d21-b98c-80e6062f3fd0.mov

Do any one know how to solve this issue ? 

React : 18.2.0
"@react-navigation/bottom-tabs": "^6.3.2",
"@react-navigation/native": "^6.0.11",
 "@react-navigation/native-stack": "^6.7.0",
"react-native": "0.70.6",

**Comment by pzatorski** [comment]
Did you find what cause this problem?
